### PR TITLE
WL 21667 - Window.browseDir() dialog doesn't let user use Esc to cancel

### PR DESCRIPTION
**Pull request by cain-kilgore**
> Follow the behavior of Window.prompt()
> [Worklist 21667](https://worklist.net/21667)

## Test Plan
1. From the Main Menu, click File and click Import Entities
2. Confirm you can press Esc to close out of it
3. Enable Advanced/Developer Menus
4. Run `Window.browseDir()` in your console and confirm you can also Esc out of that prompt.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6134/app_12121_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6134/HighFidelity-Beta-PR12121-4534419e53a6f38d8d8c0ea84c48d691774d2a5a.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6134/HighFidelity-Beta-PR12121-4534419e53a6f38d8d8c0ea84c48d691774d2a5a.exe

**Comment by QAadahlin** [comment]
Currently testing

**Comment by QAadahlin** [comment]
@conklin94122 Testing completed. PASSED.

**Comment by cain-kilgore** [comment]
@misslivirose Should be changed now :)

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6178/app_12121_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6178/HighFidelity-Beta-PR12121-057c7086374776bc75e6904e913d490ebbb62384.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6178/HighFidelity-Beta-PR12121-057c7086374776bc75e6904e913d490ebbb62384.dmg

File: interface/resources/qml/dialogs/FileDialog.qml [hunk 1]
```diff
@@ -637,10 +636,10 @@ ModalWindow {
                 case Qt.Key_Backtab:
                     event.accepted = false;
                     break;
-				case Qt.Key_Escape:
-					event.accepted = true;
-					root.click(OriginalDialogs.StandardButton.Cancel);
-					break;
+                    case Qt.Key_Escape:
```
Comment (misslivirose) [inline interface/resources/qml/dialogs/FileDialog.qml]: These four lines are indented one level too far

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6186/app_12121_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6186/HighFidelity-Beta-PR12121-104ac0ffae379d48f70b1e6827f341b83ae9177e.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6186/HighFidelity-Beta-PR12121-104ac0ffae379d48f70b1e6827f341b83ae9177e.dmg

**Pull request merged**